docs/samples/sklearn/README.md

```diff
@@ -16,7 +16,7 @@ dump(clf, 'model.joblib')
 Then, we can run the Scikit-learn Server using the generated model and test for prediction. Models can be on local filesystem, S3 compatible object storage or Google Cloud Storage.
 
 ```shell
-python -m sklearnserver --model_dir model.joblib --model_name svm
+python -m sklearnserver --model_dir file://model.joblib --model_name svm
 ```
```

Comment:
It may make sense to highlight how to use relative vs absolute path in README

Reply:
I updated the main readme docs under `python/kfserving/README.md`.